src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -66,18 +87,32 @@ public HubConnection(IConnectionFactory connectionFactory, IHubProtocol protocol
             _logger = _loggerFactory.CreateLogger<HubConnection>();
         }
 
+        /// <summary>
+        /// Starts a connection to the server.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>A <see cref="Task"/> that on completion indicates the connection is started.</returns>
         public async Task StartAsync(CancellationToken cancellationToken = default)
         {
             CheckDisposed();
             await StartAsyncCore(cancellationToken).ForceAsync();
         }
 
+        /// <summary>
+        /// Stops a connection to the server.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>A <see cref="Task"/> that on completion indicates the connection is stopped.</returns>
```

Comment:
Remarks about how the connection can be restarted by calling `StartAsync` after the `Closed` event fires